riot.py

```diff
@@ -109,3 +110,87 @@ def previous_match(name: str):
         participants.append(participants_row)
     last_match_info = pd.DataFrame(participants)
     return last_match_info
+
+
+def create_summoner_list(players_list: list, server_id: str):
+    """Gets the list of summoners and returns the information abou the summoners
+    Parameters:
+    players_list (list): list of summoner names
+    server_id (int): discord server id
+
+    Returns:
+    members_to_add (dict): summoner's latest match information
+
+    """
+
+    # dictionary for data input
+    members_to_add = {
+        server_id: [],
+    }
+
+    # accessing each player
+    for summoner in players_list:
+
+        user = watcher.summoner.by_name(MY_REGION, summoner)
+
+        user_name = user["name"]
+
+        ranked_stat = watcher.league.by_summoner(MY_REGION, user["id"])
+
+        # check if there is any rank data on user as unranked players don't have any data on ranked
+        if len(ranked_stat) > 0:
+
+            solo_rank_stat = pydash.find(ranked_stat, {"queueType": "RANKED_SOLO_5x5"})
+
+            tier_division = solo_rank_stat["tier"]
+            tier_rank = solo_rank_stat["rank"]
+
+            tier_rank_map = {"I": "1", "II": "2", "III": "3", "IV": "4"}
+
+            tier_rank_number = tier_rank_map.get(tier_rank)
+
+        # set the tier to unranked if no ranked data was found
+        else:
+            tier_division = "UNRANKED"
+            tier_rank_number = "R"
+
+        members_to_add[server_id].append(
+            {
+                "user_name": user_name,
+                "tier_division": tier_division,
+                "tier_rank_number": tier_rank_number,
+            }
+        )
+
+    return members_to_add
+
+
+def check_summoner_name(summoner: str):
+    """Checks whether the summoner exists in API
+
+    Parameters:
+    summoner (str): name of the summoner
+
+    Returns:
+    boolean (bool): if doesn't exist, boolean of whether if the summoner exists in API
+    """
+
+    try:
+        watcher.summoner.by_name(MY_REGION, summoner)
+    except requests.exceptions.HTTPError:
+        return False
+
+    return True
```

Comment:
you are not checking anything here. 

What are data needed for us to say this summoner exists in NA LOL data? 
You should check for thoese params after watcher.summoner.by_name(MY_REGION, summoner) then return false there. Right now you are just depending on league API to throw error for you.